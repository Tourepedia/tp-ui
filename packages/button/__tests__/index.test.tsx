import * as React from "react"
import * as renderer from "react-test-renderer"
import Button from "./../src/index"

describe("renders", () => {
  it("with a default type of button", () => {
    const wrapper = renderer.create(<Button />)
    expect(wrapper.toJSON()).toMatchSnapshot()
  })
})
