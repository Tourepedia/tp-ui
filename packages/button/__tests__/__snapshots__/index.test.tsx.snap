// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with a default type of button 1`] = `
<button
  type="button"
/>
`;
